src/components/TogglePinButton/TogglePinButton.tsx

```diff
@@ -11,6 +11,15 @@ import {
   TogglePinnedDocument,
 } from './TogglePinButton.generated';
 
+const useStyles = makeStyles<Theme, { pinned?: boolean }>(() => ({
+  button: ({ pinned }) => ({
+    '& > span': {
+      transition: 'transform .2s ease',
```

Comment:
Could you use a standard transition from the theme?
https://github.com/SeedCompany/cord-field/blob/b6f501a39f3420e5f6af9569dcb63c784e11dace/src/scenes/Projects/Files/DirectoryBreadcrumb.tsx#L24

There's some preset durations as well if you don't want the default.
```tsx
transition: transitions.create('transform', {
  duration: transitions.duration.short,
}),
```